style.css

```diff
@@ -257,16 +262,15 @@ main div #enlarged-image {
     top left in the enlarged image window */
 main>#enlarged-image-div>#enlarged-image-go-back-button {}
 
+/* What do we do with this? */
+
 #navigationButtons {
     width: fit-content;
     margin-right: auto;
     margin-left: auto;
     margin-bottom: 2rem;
 }
```

Comment:
^ this is old code from when we were using the arrow icons (.svg)
feel free to remove :)